docs/Users_Guide/roc_diagram.rst

```diff
@@ -82,40 +161,37 @@ the roc_diagram_defaults.yaml configuration file looks like the following:
 .. image:: roc_diagram_default.png
 
 
-To generate the above plot, use the roc_diagram_defaults.yaml and
-the empty custom configuration file, minimal_roc_diagram.yaml.
-Then, perform the following:
+Perform the following:
 
-* verify that you are running in the conda environment that
+* if using a conda environment, verify that you are running in the conda environment that
   has the required Python packages outlined in the requirements
   section
 
 * cd to the $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/roc_diagram
   directory
 
+To generate the above **"defaults"** plot (i.e using default configuration settings), use the "minimal" custom configuration file, minimal_roc_diagram.yaml.
+
 * enter the following command:
 
-  ``python roc_diagram.py ./minimal_roc_diagram.yaml``
+  ``python roc_diagram.py $WORKING_DIR/minimal_roc_diagram.yaml``
 
 
 * a `roc_diagram_default.png` output file will be created in the
-  $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/roc_diagram directory. The filename is
-  specified by the `plot_filename` value in the roc_diagram_defaults.yaml config file.
+  directory you specified in the `plot_filename` configuration setting in the `roc_diagram_defaults.yaml` config file.
 
-To generate a customized ROC diagram, use the custom_roc_diagram.yaml config
-file.
+To generate a **customized** ROC diagram (i.e. some or all default configuration settings are
+to be overridden), use the `custom_roc_diagram.yaml` config file.
 
 * enter the following command:
 
-``python roc_diagram.py ./custom_roc_diagram.yaml``
+``python roc_diagram.py $WORKING_DIR/custom_roc_diagram.yaml``
 
```

Comment:
* a `custom_roc_diagram.png` output file will be created in the
  directory you specified in the `plot_filename` configuration setting in the `custom_roc_diagram.yaml` config file.